docs/developers/content-creation/creating-a-weapon.md

```diff
@@ -1,3 +1,120 @@
 # Creating a Weapon
 
-Some Text
+When starting content creation for gmod, creating a basic weapon using the SWEP structure is one of the easiest things you can do.
+For this guide I'll be using the Visual Studio Code with the glualint extension as referenced in the "Creating an addon" section of these documents.
```

Comment:
You could also link to the 'editor' section of the 'creating an addon' page here. See my comment below for an example